README.md

```diff
@@ -124,9 +127,24 @@ Running the file using command in termainl:
 
   Easy run template for graphics.h [=> repo](https://github.com/ullaskunder3/graphics.h-project-template)
   If your dont want to write that command again and again you can create [task.json](https://github.com/ullaskunder3/graphics.h-setup/tree/main/.vscode)
-
-
-
-## Detailed Article on setting up graphics library (graphics.h) in Visual Studio Code (c++) 
-
-Detailed Article on How to setup graphics.h with Vscode and MinGW 6.3.0 Can be Found Here [How to Setup Graphics.h for Visual Studio Code.](https://github.com/sagargoswami2001/Setup-Graphics.h-for-Visual-Studio-Code)
+  
+    ## Contributors ✨
+
+Thanks goes to these wonderful people:
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/Prakash4844"><img src="https://avatars.githubusercontent.com/u/81550376?v=4" width="100px;" alt=""/><br /><sub><b>Prakash4844</b></sub></a><br />
+    <a href="#question-Prakash4844" title="Answering Questions">💬</a> <a href="https://github.com/ullaskunder3/Solution-to-graphics.h/commits?author=Prakash4844" title="Documentation">📖</a>
+  </tr>
+
+<tr>
+ <td align="center"><a href="https://github.com/sagargoswami2001"><img src="https://avatars.githubusercontent.com/u/88379870?v=4" width="100px;" alt=""/><br /><sub><b>sagargoswami2001</b></sub></a><br />
```

Comment:
It should be a continues table data td not a new tr : no problem I'll merge and fix it myself.